### ISSUE TITLE
2.x: Docs: WLS Integration Code issues

### ISSUE THREAD

**Issue by naradta** [helidon-io/helidon#5809]
[//]: # "Please remove these comments"
[//]: # "Provide a general summary of the issue in the Title above"

## Environment Details
* Helidon Version:2.5.5
* Helidon SE or Helidon MP: Helidon MP
* JDK version: JDK11
* OS: Linux
* Docker version (if applicable):19.03

----------
We have some document issues with helidon 2.x. Below are the list of issues we found while going through the quick start guide. Please help in correcting the doc.

Issues Identified in Doc:
############################
1. Seeing compilation errors with file and classpath methods in the below script. 
```
    private static Config buildConfig() {
        return Config.builder()
            .sources(
                file("/etc/config/config-file.properties").optional(), 
                classpath("META-INF/microprofile-config.properties")) 
            .build();
    }
```

    Ref page:
    https://helidon.io/docs/v2/#/mp/guides/03_config
    Integration with Kubernetes

3. Custom web context path is not working as per the steps mentioned doc.

    application.yaml content in the resources directory:
    health:
      web-context: "myhealth"  

    Ref page:
    https://helidon.io/docs/v2/#/mp/guides/04_health
    Custom Health Root Path and Port

4. Typo mistakes In Yaml format. 
    Mentioned enables inplace of enabled
  
    Ref page:
    https://helidon.io/docs/v2/#/mp/guides/05_metrics
    Controlling Metrics By Registry Type and Metric Name

5. Annotation issues
    @Path annotation is required along with @Dependent annotation for ExampleResource

    Ref Pag:
    https://helidon.io/docs/v2/#/mp/guides/07_datasource
    Inject a Datasource into Your Application Code

6. Helm repository issues are reported with helm 2.x, and no documentation is available for helm 3.x

    Ref Page:
    https://helidon.io/docs/v2/#/mp/guides/05_metrics
    Prometheus Integration

7. Class loader issues are reported with artifact helidon-integrations-cdi-jta-weld with JTA samples while trying to run the application

    Ref Page:
    https://helidon.io/docs/v2/#/mp/guides/08_jta
    Add the Helidon JTA Integration to Your Application’s Runtime Classpath

    Issue log:
    ----------
    2023.01.05 13:37:47 INFO io.helidon.common.LogConfig Thread[main,5,main]: Logging at initialization configured using classpath: /logging.properties
    Exception in thread "main" java.util.ServiceConfigurationError: Error instantiating :io.helidon.integrations.jta.cdi.NarayanaExtension
          at org.jboss.weld.util.ServiceLoader.createInstance(ServiceLoader.java:315)
    . . . 
    Caused by: java.lang.NoClassDefFoundError: javax/transaction/SystemException
            at java.base/java.lang.Class.getDeclaredConstructors0(Native Method)
    . . . 
    Caused by: java.lang.ClassNotFoundException: javax.transaction.SystemException
            at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
            at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
    . . .

**Comment by m0mus** [comment]
Please check 3.x and main too.

**Comment by ljamen** [comment]
addressed by PR #6946
